### ISSUE TITLE
Novidades de iniciativas seguidas por último deveriam aparecer primeiro

### ISSUE THREAD

**Issue by luizaugustomm** [nazareno/diferentonas-server#25]
Quando eu sigo uma nova iniciativa/cidade atualmente, as novidades dela na linha do tempo só aparecem no final da página. É mais intuitivo ordená-las para o começo.

**Comment by ricardoas** [comment]
Discordo... Isso vai quebrar o princípio de que o feed do usuário é uma linha do tempo... como não temos um esquema bacana de prioridade entre novidades (como o Facebook que prioriza os posts não lidos), acho que ordenar por tempo é a nossa opção e quebrar esse princípio só pode trazer confusão pra o cidadão ao olhar pra timeline. Se o cidadão quer ver novidades de coisas que já aconteceram numa cidade ele as verá no feed da cidade.

**Comment by nazareno** [comment]
A linha do tempo do usuário não deveria ser tal que tudo que é mais recente fique mais em cima do que o mais antigo não, @ricardoas e @luizaugustomm ?

**Comment by ricardoas** [comment]
Sim, @nazareno e estou certo de que essa é a forma implementada. Se vc seguiu uma iniciativa que só teve novidades ano passado, vc infelizmente vai ter que percorrer seu feed até a data...
